### ISSUE TITLE
Challenge turned into hell last week.

### ISSUE THREAD

**Issue by EligiusSantori** [mgp25/Instagram-API#2427]
We run a project about half a year which has nothing in common with spam and it had problems with challenge cycling time to time before but during last week no one of ~10 clients managed to finish it.

I tried to do it on my own page via command line from my own IP but result was the same: calling login() after challenge completion throws another challenge. 

I tried with different library versions (4.1, 5.0, dev); with IG_VERSION & IG_SIG_KEY constants from premium subscription; with proxy IPs; with fresh Instagram page but none of this helped.

Meanwhile official app on phone works perfectly. So I guess Instagram reinforced their antibot protection. Do anyone else ran into this problem?

**Comment by kringkaste** [comment]
Yes, we do. Same here. Since yesterday. With the existing cookies we still get responses for some requests (e.g. list all followers or medias), but for all others we only get the following error:

`No response from server. Either a connection or configuration error. {"exception":"[object] (InstagramAPI\\Exception\\EmptyResponseException(code: 0): No response from server. Either a connection or configuration error. at /local/vendor/mgp25/instagram-php/src/Client.php:469)`

After deleting the existing cookies, we also entered the loop of challenges with the same behavior.

Moving back to the old cookies, login works but same errors like before.

**Comment by kaykyr** [comment]
Same problem here!
